packages/devtools/lib/src/memory/memory.dart

```diff
@@ -185,6 +228,205 @@ class MemoryScreen extends Screen with SetStateMixin {
     return screenDiv;
   }
 
+  ClassHeapDetailStats _searchClass(String className) {
+    final List<ClassHeapDetailStats> classesData = tableStack.first.data;
+    for (ClassHeapDetailStats stat in classesData) {
+      if (stat.classRef.name == className) {
+        return stat;
+      }
+    }
+
+    return null;
+  }
+
+  void _selectClass(String className, [record = true]) {
+    final List<ClassHeapDetailStats> classesData = tableStack.first.data;
+    int row = 0;
+    for (ClassHeapDetailStats stat in classesData) {
+      if (stat.classRef.name == className) {
+        tableStack.first.selectByIndex(row, scrollBehavior: 'auto');
+        if (record) {
+          memoryPath.add(NavigationState.classSelect(className));
+        }
+        return;
+      }
+      row++;
+    }
+
+    framework.toast('Unable to find class $className', title: 'Error');
+  }
+
+  // Show the spinner we're computing...
+  Spinner createSpinner(Table<Object> table) {
+//    final html.Rectangle r = table.element.element.getBoundingClientRect();
+//    final int topPosition = r.top + (r.height / 2).round();
+//    final int leftPosition = r.left + (r.width / 2).round();
+
+    final Spinner spinner = Spinner.centered();
+//TODO(terry): Cleanup
+    //final Spinner spinner = Spinner()..clazz('padded');
+/*
+    spinner.element.style
+      ..position = 'absolute'
+      ..top = '${topPosition}px'
+      ..left = '${leftPosition}px'
+      ..zIndex = '1000';
+*/
+    return spinner;
+  }
+
+  Future<int> _selectInstanceInFieldHashCode(
+      String field, String hashCode) async {
+    final Table<Object> instanceTable = tableStack.elementAt(1);
+    final Spinner spinner = createSpinner(instanceTable);
+    instanceTable.element.add(spinner);
+
+    // There's an instances table up.
+    final List<InstanceSummary> instances = instanceTable.data;
+    int row = 0;
+    for (InstanceSummary instance in instances) {
```

Comment:
Add a TODO  to this method to find a more efficient way to do this.
For example,  Comparing by hashCode is a bit unsafe as there might be collisions and objects might not define a useful hash code. There are ways to trigger the VM to perform an object identity comparison instead of getting the hashcode or it might think sense to go straight to a direct VM API for what you need.
Ideally the VM Service would provide

Reply:
Yep I've added TODO and adding to VM Service doc as well as talking to Ben.